### ISSUE TITLE
Subset command does not produce complete font

### ISSUE THREAD

**Issue by capripio** [yeslogic/allsorts-tools#16]
I am trying to subset "OpenSans-Regular.ttf" and "Yellowtail-Regular.ttf". After sub-setting. I tried to use in browser and get following error: OTS parsing error: OS/2: missing required table.

**Comment by itanxiao** [comment]
me too

**Comment by wezm** [comment]
Yes this is a known issue, see  yeslogic/allsorts#27 for details on what need to be implemented.
